package.json

```diff
@@ -31,6 +31,7 @@
     "webpack": "2.0.6-beta"
   },
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
+    "watch-build": "webpack --watch --progress"
```

Comment:
That sounds good to me.  Go ahead and make the changes.  If everything else LGTY, then go ahead and merge.